src/util/greeting.js

```diff
@@ -1,5 +1,4 @@
 var hours = new Date().getHours();
```

Comment:
nit: Luke-warm take: Util directories are useless, they do not actually add anything to helping find the file or make it tidier :)
nit 2: if you have a single top-export,  call the file the same thing (thus: `getGreeting.js`)